tidy3d/components/data.py

```diff
@@ -1018,6 +1133,26 @@ def __getitem__(self, monitor_name: str) -> Union[Tidy3dDataArray, xr.Dataset]:
         monitor_data = self.monitor_data.get(monitor_name)
         if isinstance(monitor_data, MonitorData):
             return monitor_data.data
+        if isinstance(monitor_data, SpatialCollectionData):
+            # Unwrap symmetries
+            monitor = self.simulation.get_monitor_by_name(monitor_name)
+            sim = self.simulation
+            span_inds = sim.grid.discretize_inds(monitor.geometry, extend=True)
+            boundary_dict = {}
+            for idim, dim in enumerate(["x", "y", "z"]):
+                ind_beg, ind_end = span_inds[idim]
+                boundary_dict[dim] = sim.grid.periodic_subspace(idim, ind_beg, ind_end + 1)
+            mnt_grid = Grid(boundaries=Coords(**boundary_dict))
+            mnt_grid_dict = mnt_grid.yee.grid_dict
+            if isinstance(monitor_data, PermittivityData):
+                # Define monitor grid keys where the permittivity lives
+                mnt_grid_dict = {
+                    "eps_xx": mnt_grid_dict["Ex"],
+                    "eps_yy": mnt_grid_dict["Ey"],
+                    "eps_zz": mnt_grid_dict["Ez"],
+                }
+            return monitor_data.apply_syms(mnt_grid_dict, sim.center, sim.symmetry)
```

Comment:
can you call this with the `kwargs` instead of `args`?  Cuts down on potential bugs.

Reply:
Good point, I actually had flipped the two last arguments initially which was obviously causing wrong results